packages/collections/sort.pony

```diff
@@ -1,6 +1,21 @@
 primitive Sort[A: Seq[B] ref, B: Comparable[B] #read]
   """
   Implementation of dual-pivot quicksort.
+  
+  ## Example program
+  The following takes an reverse-alphabetical array, and sorts it alphabetically using the default String Comparator.
```

Comment:
I'd add a newline here. Generally most of the docs have an empty line after any headings.